### PR TITLE
[zh] Translate 'label-selector' into Chinese

### PR DESCRIPTION
**Pull request by twilight0620**
#32187
 Translate label-selector ( https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/label-selector/ ) into Chinese

### CODE REVIEW

**Comment by twilight0620** [comment]
Fixes #32187

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+content_type: "api_reference"
+description: "A label selector is a label query over a set of resources."
+title: "LabelSelector"
+weight: 2
+-->
+
+
+`import "k8s.io/apimachinery/pkg/apis/meta/v1"`
+
+标签选择器是对一组资源的标签查询。`matchLabels`和`matchExpressions`的结果是`ANDeD`。一个`empty`标签选择器匹配所有对象。一个`null`标签选择器不匹配任何对象。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:27]:
```suggestion
标签选择器是对一组资源的标签查询。`matchLabels`和`matchExpressions`的结果按逻辑与的关系组合。
空的标签选择器匹配所有对象。未设置的（null）标签选择器不匹配任何对象。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+   `matchExpressions` 是`LabelSelectorRequirement`的列表，要求是`ANDed`。
+   
+   <a name="LabelSelectorRequirement"></a> 
+   *标签选择器要求是包含值、键和关联键和值的运算符的选择器。*
+ 
+  <!--
+   - **matchExpressions** ([]LabelSelectorRequirement)
+   matchExpressions is a list of label selector requirements. The requirements are ANDed. 
+  *A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.*
+  -->
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:43]: The convention is to keep the English above the translated Chinese.

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+`import "k8s.io/apimachinery/pkg/apis/meta/v1"`
+
+标签选择器是对一组资源的标签查询。`matchLabels`和`matchExpressions`的结果是`ANDeD`。一个`empty`标签选择器匹配所有对象。一个`null`标签选择器不匹配任何对象。
+<!-- A label selector is a label query over a set of resources. The result of matchLabels and matchExpressions are ANDed. An empty label selector matches all objects. A null label selector matches no objects.-->
+
+<hr>
+
+- **matchExpressions** ([]LabelSelectorRequirement)
+  
+   `matchExpressions` 是`LabelSelectorRequirement`的列表，要求是`ANDed`。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:34]:
```suggestion
   `matchExpressions` 是 `LabelSelectorRequirement` 的列表，这些需求结果按逻辑与的关系来计算。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+   
+   <a name="LabelSelectorRequirement"></a>
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:36]: what's this？

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+ 
+  <!--
+   - **matchExpressions** ([]LabelSelectorRequirement)
+   matchExpressions is a list of label selector requirements. The requirements are ANDed. 
+  *A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.*
+  -->
+
+  - **matchExpressions.key** (string)， 必填
+
+    *补丁策略: 按照键`key`合并*
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:47]: 注意中英文之间留白：

```suggestion
    *补丁策略: 按照键 `key` 合并*    
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+   - **matchExpressions** ([]LabelSelectorRequirement)
+   matchExpressions is a list of label selector requirements. The requirements are ANDed. 
+  *A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.*
+  -->
+
+  - **matchExpressions.key** (string)， 必填
+
+    *补丁策略: 按照键`key`合并*    
+    
+    `key`是选择器应用的标签键.
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:49]:
```suggestion
    `key` 是选择器所适用的标签键.
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+  
+   <!-- 
+      - **matchExpressions.key** (string), required
+      *Patch strategy: merge on key `key`* 
+      key is the label key that the selector applies to.
+   -->
+   
+  - **matchExpressions.operator** (string), 必填
+ 
+    运算符表示键与一组值的关系。有效的运算符包括 `In`、`NotIn`、`Exists` 和 `DoesNotExist`。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:59]:
```suggestion
    operator 表示键与一组值的关系。有效的运算符包括 `In`、`NotIn`、`Exists` 和 `DoesNotExist`。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    
+   <!-- 
+       - **matchExpressions.operator** (string), required
+       operator represents a key's relationship to a set of values. Valid operators are In, NotIn, Exists and DoesNotExist.        
+   -->
+   
+  - **matchExpressions.values** ([]string)
+  
+    `values` 是一个字符串值数组。如果运算符为 `In` 或 `NotIn`，则 `values` 数组必须为非空。如果运算符是 `Exists` 或 `DoesNotExist`，则 `values` 数组必须为空。该数组在战略合入补丁期间被替换。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:68]: 对较长行要折行。

```suggestion
    `values` 是一个字符串值数组。如果运算符为 `In` 或 `NotIn`，则 `values` 数组必须为非空。
    如果运算符是 `Exists` 或 `DoesNotExist`，则 `values` 数组必须为空。该数组在策略性补丁
    （Strategic Merge Patch）期间被替换。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+  
+  <!--
+  - **matchExpressions.values** ([]string)
+
+    values is an array of string values. If the operator is In or NotIn, the values array must be non-empty. If the operator is Exists or DoesNotExist, the values array must be empty. This array is replaced during a strategic merge patch.
+  -->
+  
+ - **matchLabels** (map[string]string)
+  
+    `matchLabels` 是 {`key`,`value`} 键值对的映射。`matchLabels` 映射中的单个 {`key`,`value`} 键值对相当于 matchExpressions 的一个元素，其键字段为`key`，运算符为`In`，`values` 数组仅包含`value`。要求是 `ANDed`。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:78]:
```suggestion
    `matchLabels` 是 {`key`,`value`} 键值对的映射。`matchLabels` 映射中的每个 {`key`,`value`}
    键值对相当于 matchExpressions 的一个元素，其键字段为 `key`，运算符为 `In`，`values` 数组仅包含`value`。
    所表达的需求最终要按逻辑与的关系组合。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 2 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:43]: i will modify it

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 4 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md:36]: i do not really know this label, but  the English docs has it, so i left it here.

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 1 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md]: have modified.

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 3 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md]: have modified.

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 7 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md]: have modified.

File: content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md (hunk 9 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/label-selector.md]: have modified.

**Comment by twilight0620** [comment]
I have modified all issues. Thanks for your review.

**Comment by twilight0620** [comment]
/assign @tengqm 

I have modified all issues. Thanks for your review.

**Comment by twilight0620** [comment]
@tengqm hi, i have a quesion, after the pr is labeled as lgtm or approved, what else should i do to make pr merged? I found some prs with lgtm/approved label are merged just several days, but some are still not merged half years ago. What should i do to make my pr merged?

**Comment by tengqm** [comment]
@twilight0620 It is up to the reviewer's bandwidth. In my case, I'd apply lgtm and approve for trivial changes which I'm 100% sure. For bigger changes, I'd defer the lgtm or approval to another reviewer (which is the preferred practice). I'll leave an approval or lgtm when the author has made a lot of changes based on my reviews, because I don't want to be the single person guarding the zh localization. In any cases, when you feel a PR review is stalled, you can ping the zh reviewers on the slack channel.

At the same time, we encourage everyone to help review other PRs. As you can see, we have many PRs labelled 'language/zh' no reviewed for a long time. We need hands/eyes on them.

**Comment by twilight0620** [comment]
> @twilight0620 It is up to the reviewer's bandwidth. In my case, I'd apply lgtm and approve for trivial changes which I'm 100% sure. For bigger changes, I'd defer the lgtm or approval to another reviewer (which is the preferred practice). I'll leave an approval or lgtm when the author has made a lot of changes based on my reviews, because I don't want to be the single person guarding the zh localization. In any cases, when you feel a PR review is stalled, you can ping the zh reviewers on the slack channel.
> 
> At the same time, we encourage everyone to help review other PRs. As you can see, we have many PRs labelled 'language/zh' no reviewed for a long time. We need hands/eyes on them.

ok, i understand it. Thanks for your reply~

**Pull request merged**